README.md

```diff
@@ -24,6 +24,11 @@ Creating and starting containers:
 ./ccd compose up -d
 ```
 
+You must run this command once.
```

Comment:
Seems like wrong order. Modules should be enabled before first `./ccd compose up -d` is executed.

Reply:
you are correct :) i meant to add it before the compose up! will change now